### DISCUSSION TITLE
probleme to import lib.menulcd and LCD_lin3 and LCD_Config

### DISCUSSION THREAD

**Discussion by maisonnavejul** [onlaj/Piano-LED-Visualizer#277] [Q&A]
hey,
I can't import  lib.menulcd and LCD_lin3 and LCD_Config. I write in terminal for example sudo pip3 install lib.menulcd but he not find. Can you help me ?

**Comment by onlaj** [comment]
Hello.
[menulcd](https://github.com/onlaj/Piano-LED-Visualizer/blob/master/lib/menulcd.py) and  [LCD_Config ](https://github.com/onlaj/Piano-LED-Visualizer/blob/master/LCD_Config.py) are in repository

**Reply by maisonnavejul** [reply]
thanks, so who i solve this problemes (sorry for my english, i'm french)
![image](https://user-images.githubusercontent.com/92145165/142266387-04ca6dcd-4055-4b4f-8ed2-dbe32e84289d.png)

**Comment by onlaj** [comment]
Did you enable [SPI](https://www.raspberrypi-spy.co.uk/2014/08/enabling-the-spi-interface-on-the-raspberry-pi/)?

**Reply by maisonnavejul** [reply]
i just use putty and WinSCP

**Reply by onlaj** [reply]
So type
`sudo raspi-config` in console and follow the instructions from the link

**Reply by maisonnavejul** [reply]
![image](https://user-images.githubusercontent.com/92145165/142267419-df7dfeea-d89a-4555-bd2e-3e8e16ce0239.png)

**Reply by maisonnavejul** [reply]
thanks, it work ! 
but i have a another error
![image](https://user-images.githubusercontent.com/92145165/142268308-a7f12ae0-3386-40c6-b9d1-2bb7808a1487.png)

**Reply by onlaj** [reply]
This is indeed an error on my side, this file is optional and shouldn't be required on launch.
Follow this part of manual: https://github.com/onlaj/Piano-LED-Visualizer/blob/master/Docs/manual_installation.md#2-creating-autoconnect-script

**Reply by maisonnavejul** [reply]
thank you so much for the responsivenes I will try

**Comment by maisonnavejul** [comment]
it work now, thanks so much for your help and your work
